src/OpenLoco/src/Ui/Windows/MusicSelection.cpp

```diff
@@ -99,20 +101,20 @@ namespace OpenLoco::Ui::Windows::MusicSelection
                 break;
             }
 
-            StringId text_colour_id = StringIds::black_stringid;
+            StringId textColour = StringIds::black_stringid;
```

Comment:
text colour to me sounds like the colour of the text but this is just a string with a colour. Now i know that sounds like the exact same thing but think it would make more sense to be called `colouredText`.

Reply:
> text colour to me sounds like the colour of the text but this is just a string with a colour. Now i know that sounds like the exact same thing but think it would make more sense to be called `colouredText`.

I think that makes sense - this is a piece of text that describes colour, not a colour that describes a piece of text

Reply:
🤔 Maybe call it 'colouring', since I think that is moreso what it does... `textColouring`? `colouringStringId`?